workflows/process/guppy.nf

```diff
@@ -36,7 +36,18 @@ process guppy_gpu {
     output:
         tuple val(name), path("*.fastq.gz"), emit: reads
         tuple val(name), path("fastq_tmp/*.txt"), emit: summary
-    script:
+    script:       
+        if (params.rapid) {
+            guppy_arrangement_files = 'barcode_arrs_rbk4.cfg'
```

Comment:
@DataSpott this needs the rbk file that would support 96 barcodes (check newest guppy)